app/src/main/AndroidManifest.xml

```diff
@@ -1,10 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="com.example.filemanager">
 
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <uses-permission
+        android:name="android.permission.MANAGE_EXTERNAL_STORAGE"
```

Comment:
For me it works without this permission as well, don't know why. Can you try without it? just use `WRITE_EXTERNAL_STORAGE`.

Reply:
Do you mean READ_EXTERNAL_STORAGE or MANAGE_EXTERNAL_STORAGE?

Reply:
Can remove both of them and try, just keep the `WRITE_EXTERNAL_STORAGE` only.